ziti/edge/network/conn.go

```diff
@@ -51,7 +51,7 @@ type edgeConn struct {
 	edge.MsgChannel
 	readQ                 *noopSeq[*channel.Message]
 	inBuffer              [][]byte
-	msgMux                edge.MsgMux
+	msgMux                edge.ConnMux[any]
```

Comment:
Nothing in the SDK is currently using the generic data for each conn, so I put `any` in. However, IIRC there was a future use case for this to add context data for reporting/debugging.